packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/GoogleMapMarkerController.m

```diff
@@ -225,163 +332,220 @@ @interface FLTMarkersController ()
 
 @property(strong, nonatomic) NSMutableDictionary *markerIdentifierToController;
 @property(strong, nonatomic) FlutterMethodChannel *methodChannel;
+@property(strong, nonatomic) UIImage *emptyClusterMarker;
+@property(strong, nonatomic) NSString *fontPath;
 @property(weak, nonatomic) NSObject<FlutterPluginRegistrar> *registrar;
 @property(weak, nonatomic) GMSMapView *mapView;
 
+
 @end
 
 @implementation FLTMarkersController
 
 - (instancetype)initWithMethodChannel:(FlutterMethodChannel *)methodChannel
                               mapView:(GMSMapView *)mapView
                             registrar:(NSObject<FlutterPluginRegistrar> *)registrar {
-  self = [super init];
-  if (self) {
-    _methodChannel = methodChannel;
-    _mapView = mapView;
-    _markerIdentifierToController = [[NSMutableDictionary alloc] init];
-    _registrar = registrar;
-  }
-  return self;
+    self = [super init];
+    if (self) {
+        _methodChannel = methodChannel;
+        _mapView = mapView;
+        _markerIdentifierToController = [[NSMutableDictionary alloc] init];
+        _registrar = registrar;
+        _emptyClusterMarker = [self baseClusterMarker];
+        _fontPath = [self loadFont];
+    }
+    return self;
+}
+
+
+void CFSafeRelease(CFTypeRef cf) {
+    if (cf != NULL) {
+        CFRelease(cf);
+    }
+}
+
+-(NSString *)loadFont {
+    NSBundle *frameworkBundle = [NSBundle bundleForClass:self.classForCoder];
+    NSURL *bundleURL = [[frameworkBundle resourceURL] URLByAppendingPathComponent:@"CozyFonts.bundle"];
+    NSBundle *bundle = [NSBundle  bundleWithURL:bundleURL];
+    NSURL *fontURL = [bundle URLForResource:@"oatmealpro2_semibold" withExtension:@"otf"];
+    NSData *inData = [NSData dataWithContentsOfURL:fontURL];
+    CFErrorRef error;
+    CGDataProviderRef provider = CGDataProviderCreateWithCFData((CFDataRef)inData);
+    CGFontRef font = CGFontCreateWithDataProvider(provider);
+    if (!CTFontManagerRegisterGraphicsFont(font, &error)) {
+        CFStringRef errorDescription = CFErrorCopyDescription(error);
+        NSLog(@"Failed to load font: %@", errorDescription);
+        CFRelease(errorDescription);
+    }
+    NSString *fontName = (__bridge NSString *)CGFontCopyPostScriptName(font);
+    CFSafeRelease(font);
+    CFSafeRelease(provider);
+    return fontName;
+}
+
+
+-(UIImage *)baseClusterMarker {
+    CGFloat size = 66;
```

Comment:
When we deal with the UI in a feature, it's always a good idea to include screenshots in the PR.
Because there are so many fixed sizes in this PR, could you include screenshots for the smallest iPhone we support, the biggest and maybe one for the iPad too? (While we don't actively support the iPad, Apple Reviewers almost always use them to review our app, so it's a good idea to check if it looks OK there too)

Reply:
Will do. Currently having some troubles opening the app because of getIt. But as soon as it opens i'll put on some screenshots.

Reply:
Just posted the images in the other PR.